### DISCUSSION TITLE
Problem with setup - OSError: [Errno 30] Read-only file system

### DISCUSSION THREAD

**Discussion by YourGod4** [davidrapan/ha-solarman#63] [Q&A] [answered]
i dont know how solve this error, i think i need change read only file format do i?

![inverter log](https://github.com/user-attachments/assets/89ff121d-05c0-4b9b-9cb9-9426c707fbf9)
![iverter 1](https://github.com/user-attachments/assets/e5d70ddb-7d1b-4870-bade-6e81374c2455)

**Comment by davidrapan** [comment, marked as answer]
https://github.com/davidrapan/ha-solarman/issues/39

**Comment by davidrapan** [comment]
You can try running `ha core rebuild` to fix the issue.
